app/src/androidTest/java/com/github/se/signify/ui/screens/home/HomeScreenTest.kt

```diff
@@ -91,51 +93,59 @@ class HomeScreenTest {
   // The following 2 tests should be moved to their own file.
   @Test
   fun exerciseListDisplaysExerciseButtons() {
-    val exercises = listOf(Exercise("Easy"), Exercise("Medium"), Exercise("Hard"))
+    val exercises =
+        listOf(
+            ExerciseInformation(ExerciseLevel.Easy),
+            ExerciseInformation(ExerciseLevel.Medium),
+            ExerciseInformation(ExerciseLevel.Hard))
 
     composeTestRule.setContent { ExerciseList(exercises, navigationActions) }
 
     exercises.forEach { exercise ->
       composeTestRule
-          .onNodeWithTag("${exercise.name}ExerciseButton")
+          .onNodeWithTag("${exercise.level}ExerciseButton")
           .performScrollTo()
           .assertIsDisplayed()
     }
   }
 
   @Test
   fun exerciseButtonTextDisplaysCorrectly() {
-    val exercises = listOf(Exercise("Easy"), Exercise("Medium"), Exercise("Hard"))
+    val exercises =
+        listOf(
+            ExerciseInformation(ExerciseLevel.Easy),
+            ExerciseInformation(ExerciseLevel.Medium),
+            ExerciseInformation(ExerciseLevel.Hard))
 
     composeTestRule.setContent { ExerciseList(exercises, navigationActions) }
 
     exercises.forEach { exercise ->
       composeTestRule
-          .onNodeWithTag("${exercise.name}ExerciseButtonText", useUnmergedTree = true)
+          .onNodeWithTag("${exercise.level}ExerciseButtonText", useUnmergedTree = true)
           .performScrollTo()
           .assertIsDisplayed()
-          .assertTextEquals(exercise.name)
+          .assertTextEquals(exercise.level)
     }
   }
 
   @Test
   fun clickingExerciseButtonsCallsOnClick() {
     val exercises =
         listOf(
-            Exercise("Easy", "Exercise Screen Easy"), Exercise("Hard", "Exercise Screen Hard")
-            // Need to modify hard to medium and hard should be as another exercise
-            )
+            ExerciseInformation(ExerciseLevel.Easy),
+            ExerciseInformation(ExerciseLevel.Medium),
+            ExerciseInformation(ExerciseLevel.Hard))
```

Comment:
## Requested changes

Use `enum.values()` here.

Reply:
I will take into account this change, but rather use `ExerciseLevel.entries`.